lib/tasks/run.rake

```diff
@@ -17,7 +17,9 @@
 #++
 
 def set_y2dir
-  ENV["Y2DIR"] = Dir["**/src"].join(":")
+  dirs = Dir["**/src"]
+  dirs << ENV["Y2DIR"] if ENV["Y2DIR"] && !ENV["Y2DIR"].empty?
```

Comment:
this is wrong, as Y2DIR can already contain ":" so it should look like
```
dirs.concat(ENV["Y2DIR"].split(":")) if ENV["Y2DIR"] && !ENV["Y2DIR"].empty?
```

and you can simplify it with to_s so in the end it can look like
```
ENV["Y2DIR"] = (Dir["**/src"] + ENV["Y2DIR"].to_s.split(":")).join(":")
```

Reply:
The already existing `:` should be OK.

["foo", "/already:/existing"].join(":") => "foo:/already:/existing"

I wanted to avoid the unnecessary parsing and merging the old value.

Reply:
ok, so lets use it.